contents.json

```diff
@@ -30207,6 +30226,22 @@
       "suggested_by": "@dkhamsing",
       "stars": 230
     },
+    {
+      "title": "Space Empires",
```

Comment:
Can you remove the space empires entry ?

Reply:
OHH still there!!

Reply:
done

Reply:
Done where? What is the commit